pkg/sysregistriesv2/testdata/missing-mirror-location.conf

```diff
@@ -1,10 +0,0 @@
-unqualified-search-registries = ["registry-a.com"]
-
-[[registry]]
-location = "registry-a.com"
-
-[[registry]]
-location = "registry-b.com"
-[[registry.mirror]]
-location = "mirror-b.com"
-[[registry.mirror]]
```

Comment:
Why are these files removed?

Reply:
let me check, I think presence of that file caused that test to fail.

Reply:
That would be concerning since a Mirror should always have a location.

Reply:
check for empty mir.Location added to postProcessRegistries. Will push in the next commit.

Reply:
@vrothberg This is the fallout of https://github.com/containers/image/pull/1191#discussion_r610059666 , allowing empty values in `parseLocation`.

Reply:
added check for empty mir.Location. PTAL.